history/characters/1_azerothian.txt

```diff
@@ -13158,8 +13156,6 @@
 			set_variable = { name = wc_dexterity_physical_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_4_physical_trait_value }
 			set_variable = { name = wc_strength_physical_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_4_physical_trait_value }
 		}
-		# trait=physical_lifestyle_dexterity_4
-		# trait=physical_lifestyle_strength_4
```

Comment:
I don't think you ought to remove those

Reply:
ok, doesnt look like this system is being used at all. idk how it made it into production. anyway i will readd them